### PR TITLE
feat: add Storage contract deployment to Dev env.

### PR DESCRIPTION
**Pull request by julianmrodri**
PR - This setups the Storage contract in the Dev Environment.

* Modifies `install.sh` and `deploy.sh` scripts to accept "rns" and "storage" parameters in the command line to setup services independently.
* Setups and Deploys the Storage Marketplace Contract (./storage-dev) - and creates config file with the Contract Address.
* Updates `Readme.md` with new instructions for RNS (too early to complete the Storage part).
* Creates combined config files for Cache and UI in `out` folder based on the selected service.

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -1,9 +1,20 @@
...
+
+for i in "$@"
+do
+   if [ $i == "rns" ] || [ $i == "storage" ] ;  then
+        SERVICE_DIR="./"$i"-dev"
+        BASE_DIR=$(pwd)
+   
+        cd "$SERVICE_DIR"
+        rm -rf node_modules package-lock.json
+        npm i
```
Comment (vojtechsimetka) [inline install.sh:16]: I guess we should actually make this `npm ci`
```suggestion
        npm ci
```

File: README.md [hunk 2]
```diff
@@ -54,15 +54,15 @@ The Ganache blockchain will now run and it is available to deploy the correspond
 
 
 ### 1.2. Deploying smart contracts
-First, install the dependencies (make sure to **use node v10**, you can switch using `nvm use 10`)
+First, install the RNS dependencies (make sure to **use node v10**, you can switch using `nvm use 10`). 
 
 ```
-sh install.sh
+sh install.sh rns
```
Comment (vojtechsimetka) [inline README.md:60]:
```suggestion
sh install.sh rns storage
```

File: README.md [hunk 3]
```diff
@@ -54,15 +54,15 @@ The Ganache blockchain will now run and it is available to deploy the correspond
...
 ```
 
 Run the deployment script for the RNS and Marketplace contracts deploying to ganache network
 ```
-sh deploy.sh
+sh deploy.sh rns
```
Comment (vojtechsimetka) [inline README.md:65]:
```suggestion
sh deploy.sh rns storage
```

File: README.md [hunk 4]
```diff
@@ -54,15 +54,15 @@ The Ganache blockchain will now run and it is available to deploy the correspond
 
 
 ### 1.2. Deploying smart contracts
-First, install the dependencies (make sure to **use node v10**, you can switch using `nvm use 10`)
+First, install the RNS dependencies (make sure to **use node v10**, you can switch using `nvm use 10`).
```
Comment (vojtechsimetka) [inline README.md:57]:
```suggestion
First, install the dependencies (make sure to **use node v10**, you can switch using `nvm use 10`). 
```

File: install.sh (hunk 1 above)
Comment (julianmrodri) [reply install.sh:18]: It didnt work due to a requirement on the package-lock.json lockfile version. I will leave it as-is for now as it regenerates the package-lock.json anyway.   

`npm ERR! cipm can only install packages with an existing package-lock.json or npm-shrinkwrap.json with lockfileVersion >= 1. Run an install with npm@5 or later to generate it, then try again.`

**Pull request merged**